src/pods/employee/components/report.validations.ts

```diff
@@ -0,0 +1,21 @@
+import { ValidationSchema, Validators } from '@lemoncode/fonk';
+import { createFormikValidation } from '@lemoncode/fonk-formik';
+
+const validationSchema: ValidationSchema = {
+  field: {
+    month: [
+      {
+        validator: Validators.required,
+        message: 'Campo obligatorio',
+      },
+    ],
+    year: [
+      {
+        validator: Validators.required,
+        message: 'Campo obligatorio',
```

Comment:
There's a way to create unique messages for each validator no need to override per validationSchema

check this: campus-admin/src/core/i18n.ts

And check this: https://lemoncode.github.io/fonk-doc/messages/internationalization

Section "Global Error Messages"

Reply:
Done. I did not create, global validations, because it have been created allready.